java/kafka/consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -30,11 +45,21 @@ public static void main(String[] args) {
         Properties props = KafkaConsumerConfig.createProperties(config);
         int receivedMsgs = 0;
 
-        if (System.getenv("JAEGER_SERVICE_NAME") != null)   {
-            Tracer tracer = Configuration.fromEnv().getTracer();
-            GlobalTracer.registerIfAbsent(tracer);
+        TracingSystem tracingSystem;
+        tracingSystem = TracingSystem.forValue(config.getTracingSystem());
+
+        if (tracingSystem != null) {
+            if (tracingSystem == KafkaConsumerExample.TracingSystem.JAEGER) {
+                Tracer tracer = Configuration.fromEnv().getTracer();
+                GlobalTracer.registerIfAbsent(tracer);
 
-            props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, TracingConsumerInterceptor.class.getName());
+                props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, io.opentracing.contrib.kafka.TracingConsumerInterceptor.class.getName());
```

Comment:
We should keep the import and use the short name instead of this.

Reply:
Te reason the import prefix is added here instead of keeping the import is: Jaeger (opentracing) and opentelemetry use the same class name and the only way i could differencieate between the two is to add the import in the lines of code. Is there another way to do this? Thanks. See below:
` props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG,io.opentracing.contrib.kafka.TracingConsumerInterceptor.class.getName());`
`props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG,io.opentelemetry.instrumentation.kafkaclients.TracingConsumerInterceptor.class.getName());`